mastodon/src/main/java/org/joinmastodon/android/ui/utils/UiUtils.java

```diff
@@ -1341,10 +1341,6 @@ public static void openURL(Context context, String accountID, String url) {
 		openURL(context, accountID, url, true);
 	}
 
-	public static void openURL(Context context, String accountID, String url, Object parentObject) {
```

Comment:
Why did you remove this? I think it was uptream that added it, so maybe we should keep it just in case.

Reply:
It's not used anywhere and hasn't been for awhile

Reply:
Sure... In case upstream needs it we will readd it. But this should be 2 commits instead of one.